CHANGELOG.rst

```diff
@@ -0,0 +1,9 @@
+=========
+CHANGELOG
+=========
+
+
+0.1.0 (UNRELEASED)
```

Comment:
the nice thing is that it fits with date length:

```
2016-01-30
UNRELEASED
```

so it has to be exactly 10 letters. In addition we could also thing about the date format.